packages/experimental/RadioGroup/src/__tests__/__snapshots__/RadioExperimental.test.tsx.snap

```diff
@@ -59,10 +59,10 @@ exports[`Radio component tests Radio default 1`] = `
         "borderWidth": 1,
         "height": 20,
         "justifyContent": "center",
-        "marginBottom": undefined,
-        "marginLeft": 8,
-        "marginRight": undefined,
-        "marginTop": undefined,
+        "marginBottom": 6,
+        "marginLeft": 4,
```

Comment:
Take a look to confirm if this is an expected change with snapshots, or it's a mapping issue.